webapp/color_and_brightness.py

```diff
@@ -21,15 +22,19 @@
 if TYPE_CHECKING:
     import astral.location
 
-# Same as homeassistant.const.SUN_EVENT_SUNRISE and homeassistant.const.SUN_EVENT_SUNSET
-# We re-define them here to not depend on homeassistant in this file.
-SUN_EVENT_SUNRISE = "sunrise"
-SUN_EVENT_SUNSET = "sunset"
 
-SUN_EVENT_NOON = "solar_noon"
-SUN_EVENT_MIDNIGHT = "solar_midnight"
+class SunEvent(str, Enum):
```

Comment:
Can you import this from the `custom_components` folder instead of duplicating it?